lightning-network-book.tex

```diff
@@ -177,138 +176,138 @@ \subsection{Electronic coins as chains of digital signatures}
+But even in that case it would not be feasible\footnote{actually it would be more accurate to say that it is highly  unlikely} to compute the solution with one computer or even a cluster of computers within a reasonable time interval.
 \item \textbf{Encryption:} With the help of the public key one can easily encrypt a message.
 It is hard to decrypt the message with the public key.
 But the encrypted message can easily be decrypted with the help of the private key.
-Remember the private key is hard to derive from the pulic key.
+Remember the private key is hard to derive from the public key.
 So we already see where the names come from.
 A person can publish their public key and everyone else can easily compute an encrypted message.
-This message can only easily be accessed (decrypted) if the private key is known.
+This message can only be easily accessed (decrypted) if the private key is known.
 \item \textbf{Signatures:} Given a piece of information and a private key one can easily compute a digital signature.
-  Any person who has access to this piece of information, the signature and the public key can easily verify that this signature was computeded with the private key from that particular piece of information.
-  If only a single bit in the information was changed after the signature was computed it cannot be verified with the public key anymore.
-  As with the first two properties digital signatures are asymmetric in the sense that it is very hard to produce a valid signature for a message and a private key if only the public key is known.
-  It is considered save to assume that the owner of the private (and thus also the public) key is indeed the authorized person to have created that message.
-  Due to the asymmetric nature and the computanional hardness it is almost impossible to forge a digital signature.
+  Any person who has access to the piece of information, the signature and the public key can easily verify that this signature was computed with the private key from that particular piece of information.
+  If only a single bit in the information was changed after the signature was computed, the verification can no longer be completed using the public key.
+  As with the first two properties, digital signatures are asymmetric in the sense that it is easy to produce a valid signature for a message if the private key is known, but very hard to forge the signature if only the public key is known.
+  It is considered safe to assume that the owner of the private key (and thus also the public key) is indeed the person authorized to have created that message.
+  Due to the asymmetry and the computational hardness, it is almost impossible to forge a digital signature.
```

Comment:
kind of a doubling. as the hardness to forge someting was already mentioned two sentences before

Reply:
True, I will make a change to include only that statement only once.